src/java/picard/analysis/directed/CalculateHsMetrics.java

```diff
@@ -24,75 +24,22 @@
-import java.util.TreeSet;
-
 /**
  * Calculates a set of HS metrics from a sam or bam file.  See HsMetricsCollector and CollectTargetedMetrics for more details.
  *
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Calculates a set of Hybrid Selection specific metrics from an aligned SAM" +
+        usage = "DEPRECATED: Use CollectHsMetrics instead. Calculates a set of Hybrid Selection specific metrics from an aligned SAM" +
                 "or BAM file. If a reference sequence is provided, AT/GC dropout metrics will " +
                 "be calculated, and the PER_TARGET_COVERAGE option can be used to output GC and " +
                 "mean coverage information for every target.",
-        usageShort = "Calculates Hybrid Selection-specific metrics for a SAM or BAM file",
+        usageShort = "DEPRECATED: Use CollectHsMetrics instead.",
         programGroup = Metrics.class
 )
-public class CalculateHsMetrics extends CollectTargetedMetrics<HsMetrics, HsMetricCollector> {
-
-    @Option(shortName = "BI", doc = "An interval list file that contains the locations of the baits used.", minElements=1)
-    public List<File> BAIT_INTERVALS;
-
-    @Option(shortName = "N", doc = "Bait set name. If not provided it is inferred from the filename of the bait intervals.", optional = true)
-    public String BAIT_SET_NAME;
-
-    @Override
-    protected IntervalList getProbeIntervals() {
-        for (final File file : BAIT_INTERVALS) IOUtil.assertFileIsReadable(file);
-        return IntervalList.fromFiles(BAIT_INTERVALS);
-    }
-
-    @Override
-    protected String getProbeSetName() {
-        if (BAIT_SET_NAME != null) {
-            return BAIT_SET_NAME;
-        } else {
-            final SortedSet<String> baitSetNames = new TreeSet<String>();
-            for (final File file : BAIT_INTERVALS) {
-                baitSetNames.add(CollectTargetedMetrics.renderProbeNameFromFile(file));
-            }
-            return StringUtil.join(".", baitSetNames);
-        }
-    }
-
-    /** Stock main method. */
-    public static void main(final String[] argv) {
-        System.exit(new CalculateHsMetrics().instanceMain(argv));
-    }
 
-    @Override
-    protected HsMetricCollector makeCollector(final Set<MetricAccumulationLevel> accumulationLevels,
-                                              final List<SAMReadGroupRecord> samRgRecords,
-                                              final ReferenceSequenceFile refFile,
-                                              final File perTargetCoverage,
-                                              final IntervalList targetIntervals,
-                                              final IntervalList probeIntervals,
-                                              final String probeSetName) {
-        return new HsMetricCollector(accumulationLevels, samRgRecords, refFile, perTargetCoverage, targetIntervals, probeIntervals, probeSetName);
-    }
-}
+@Deprecated
```

Comment:
I think it would be good to override the parameters needed to make the default behavior of CalculateHsMetrics remain unchanged. Is that possible?